packages/cashier/src/components/no-balance/no-balance.tsx

```diff
@@ -7,17 +7,25 @@ import { localize, Localize } from '@deriv/translations';
 import { useStore, observer } from '@deriv/stores';
 import { useCashierStore } from '../../stores/useCashierStores';
 
-const NoBalance = observer(({ history }: RouteComponentProps) => {
+type TNoBalanceProps = RouteComponentProps & {
+    onClickDeposit?: () => void;
+};
+
+const NoBalance = observer(({ history, ...props }: TNoBalanceProps) => {
```

Comment:
@sergei-deriv Could you destructure the prop here instead of passing `...props`🙏🏻

Reply:
Done